nasa/earthdata-varinfo
# Use CF-Convention long_name in UMM-Var.

## Description

This PR updates the generation of the `LongName` attribute in UMM-Var records to preferentially use the `long_name` CF-Convention attribute if it is present for a variable.

Due to this change of definition in `LongName`, the native ID for the variable has been changed from `<collection concept ID>-<variable LongName>` to `<collection concept ID>-<variable Name>`. This change will have no impact on previously generated records as the `Name` and `LongName` fields generated were previously identical.

## Local Test Steps

* Pull this version of the code and install it with `pip install -e .`
* Run the UMM-Var generation routine on a collection with variables that have a `long_name` attribute. Check that value is used in place of the variable path.

## PR Acceptance Checklist
